tests/candev/Makefile

```diff
@@ -7,7 +7,36 @@ USEMODULE += isrpipe
 # define the CAN driver you want to use here
 CAN_DRIVER ?= native
 
-ifeq ($(CAN_DRIVER), PERIPH_CAN)
+ifeq ($(CAN_DRIVER), MCP2515)
+    FEATURES_REQUIRED = periph_gpio periph_spi periph_gpio_irq
+
+    USEMODULE += mcp2515
+
+	#set device paramters
+	TEST_MCP2515_SPI   ?= SPI_DEV\(0\)
+	TEST_MCP2515_SPI_CLK   ?= SPI_CLK_1MHZ
+	TEST_MCP2515_SPI_MODE   ?= SPI_MODE_0
+	TEST_MCP2515_CS    ?= GPIO_PIN\(PORT_B,0\)
+	TEST_MCP2515_RESET ?= GPIO_PIN\(PORT_B,2\)
+	TEST_MCP2515_INT ?= GPIO_PIN\(PORT_B,1\)
+	TEST_MCP2515_CLK ?=\(8000000ul\)
+	TEST_MCP2515_BITRATE ?= \(125000\)
+
+	#INT_PIN = PB3 --> requires AVR bank1 on avr. Change accordingly to used hardware.
+	USEMODULE += atmega_pcint0
```

Comment:
I have no idea how the AVRs work and I'm actually a bit surprised to see this platform specific dependency. However, if it is really necessary, it should be included as a platform specific dependency 
```
ifneq (,$(filter <avr board>,$(BOARD)))
...
endif
```

Reply:
Problem is that it's not only platform dependent, but also user (or board) dependant. 
For example, if I want to use it on PB1 on an atmega1284p, I need to enable pcint1. If I want to use the same PB1 on an atmega328p, I need to enable pcint0. Or if I hook it up to PD1, I'll need pcint2. 
It's unpredictable really... It all depends on what target the user has and how he has everything wired. 
So personally I feel like it has to go in the Makefile, because it is something the user needs to be able to adjust to his own set-up. But still... it does look ugly to put it there...

Reply:
Sure, it is wiring dependent (like the other pins too) and usually we don't merge any of these configurations to master, except there is a "hard wired" board. In that case, we define the configuration in the board.h file. 
Now with `atmega_pcint0` I really don't know how this is handled. For instance, if you connect an external radio (e.g., mrf24j40) it requires an interrupt pin to inform the MCU – same story. How do we deal with that in the current state?

Reply:
I'll remove it entirely for now, but if anyone has better suggestions: they are very welcome! :)